crates/scheduler/src/call/crypto/ec.rs

```diff
@@ -39,7 +39,7 @@ pub fn process<B: Board>(call: Api<DispatchSchedulerCall<B>>) {
 
 fn is_supported<B: Board>(call: SchedulerCall<B, api::is_supported::Sig>) {
     let api::is_supported::Params { curve } = call.read();
-    call.reply(convert_curve::<B>(*curve).map(|x| Ok(x.is_ok())))
+    call.reply(try { convert_curve::<B>(*curve).map(|x| x.is_ok())? })
```

Comment:
nit: by moving the `?` we can skip the `map()`
```suggestion
    call.reply(try { convert_curve::<B>(*curve)?.is_ok() })
```